src/config-loader.js

```diff
@@ -0,0 +1,188 @@
+// This file is a rewrite of `@sveltejs/vite-plugin-svelte` without the `Vite`
+// parts: https://github.com/sveltejs/vite-plugin-svelte/blob/e8e52deef93948da735c4ab69c54aced914926cf/packages/vite-plugin-svelte/src/utils/load-svelte-config.ts
+import { logger } from '@storybook/client-logger';
+import { createRequire } from 'module';
+import path from 'path';
+import fs from 'fs';
+import { pathExists } from "fs-extra";
+import { pathToFileURL, fileURLToPath } from 'url';
+
+/**
+ * Try find svelte config and then load it.
+ *
+ * @returns {import('@sveltejs/kit').Config | undefined}
+ * Returns the svelte configuration object.
+ */
+export async function loadSvelteConfig() {
+  const configFile = await findSvelteConfig();
+
+  // no need to throw error since we handle projects without config files
+  if (configFile !== undefined) {
```

Comment:
```suggestion
  if (configFile === undefined) {
```

A pretty critical typo 😅